roles/cis/tasks/3/3.5.1.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: "3.5.1 Ensure DCCP is disabled (Not Scored)"
+  vars:
+   modprobe_conf: /etc/modprobe.d/modprobe.conf
+  block:
+    - name: "verify if modprobe.conf is present"
+      shell: |
+        if [ ! -f "{{ modprobe_conf }}" ]; then
+            if [ ! -d "/etc/modprobe.d" ]; then
+                mkdir /etc/modprobe.d
+            fi
+            touch {{ modprobe_conf }}
+        fi
+    - name: "3.5.1 write DCCP setting to modprobe.conf"
+      shell: |
+        LINE='install dccp /bin/true'
+        FILE="{{ modprobe_conf }}"
+        grep -qF -- "$LINE" "$FILE" || echo "$LINE" >> "$FILE"
```

Comment:
can you look at `lineinfile` module for this?

Reply:
Fixed